StudCity.Frontend/src/app/features/chatsSlice.js

```diff
@@ -50,7 +50,24 @@ export const fetchUserChats = createAsyncThunk(
 		}
 	}
 );
-
+export const createChat = createAsyncThunk(
+	'room/createChat',
+	async (
+		chatData,
+		{ fulfillWithValue, rejectWithValue },
+		roomService = new RoomService()
+	) => {
+		try {
+			const data = await roomService.createChat(chatData);
+			return fulfillWithValue(data);
+		} catch (err) {
+			if (!err.response) {
```

Comment:
What about adding **error** `showAlert ` in the catch statement? Also, maybe add a success alert when creating is successful

Reply:
is it required for this functionality? @maryanchuk04